ci-operator/step-registry/ovn/sdn-migration-rollback/ovn-sdn-migration-rollback-commands.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET=${TARGET:-OpenShiftSDN}
+
+oc patch MachineConfigPool master --type='merge' --patch '{"spec":{"paused":true}}'
+oc patch MachineConfigPool worker --type='merge' --patch '{"spec":{"paused":true}}'
+
+# Reset the spec.migration before we can set it to other value
+oc patch Network.operator.openshift.io cluster --type='merge' --patch '{"spec":{"migration":null}}'
+# Wait until CNO update the applied-cluster cm to the latest
+sleep 10
+
+# Allow rollback
+# Change network to target network in Network.config.openshift.io the CR to trigger machine config update by MCO.
+oc patch Network.operator.openshift.io cluster --type='merge' --patch "{\"spec\":{\"migration\":{\"networkType\":\"${TARGET}\"}}}"
+oc patch Network.config.openshift.io cluster --type='merge' --patch "{\"spec\":{\"networkType\":\"${TARGET}\"}}"
+
+sleep 30
+# Wait until the multus pods are restarted
+timeout 300 oc rollout status ds/multus -n openshift-multus
+
+# Reboot all the nodes
+
+oc get pod -n openshift-machine-config-operator | grep daemon|awk '{print $1}'|xargs -i oc rsh -n openshift-machine-config-operator {} chroot /rootfs shutdown -r +1
+sleep 60
```

Comment:
Get a list of nodes and wait for all of them to become NotReady and then Ready

Reply:
As we will lose connection to the API server, I don't think we can see all of them become NotReady. But we wait for them to become Ready after the reboot.

Reply:
fixed.

Reply:
API service is LB-backed, connection may restart but `oc wait` should be handling that afaik

Reply:
All the nodes are rebooted at the same time, so lB doesn't help here.